dropwizard-testing/src/test/resources/logback-test.xml

```diff
@@ -1,10 +1,3 @@
 <configuration>
-    <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
-        <encoder>
-            <pattern>%d{HH:mm:ss.SSS} [%thread] %-5level %logger{36} - %msg%n</pattern>
-        </encoder>
-    </appender>
-    <root level="info">
-        <appender-ref ref="STDOUT"/>
-    </root>
+    <root level="off"/>
```

Comment:
Logging would be disabled hee, although it was enabled perviously.

Reply:
Introduced 7 years ago in 14bbdada40fb680095797d8ee822c56eb1fa2ed9 which enabled running tests in parallel. I suspect this was done to make it easier to see what was going on with the logging acquisition spinlock etc.

Reply:
Yes, seems to be enabled for debugging purposes.

At the moment, the logging configuration for `dropwizard-testing` will probably get overridden fast as described in #5479. So it won't be that important if we have logging enabled or disabled, since the `BootstrapLogging.bootstrap()` method will enable the Dropwizard defaults at the first integration test.